backend/elastic.js

```diff
@@ -105,6 +105,22 @@ class Elastic {
     }, {});
   }
 
+  async getSubjectsFromClasses() {
```

Comment:
There should be an easy aggregation to grab the subjects that we can then call on server startup, and then from there check against it when we need to, but doing some wrestling with ES right now to get it right. WIP for now